docs/releases/1.24-NOTES.md

```diff
@@ -24,8 +26,6 @@ This is a document to gather the release notes prior to the release.
 
 * The `node-role.kubernetes.io/master` and `kubernetes.io/role` labels are deprecated and might be removed from control plane nodes in future versions of kOps.
 
-* Due to lack of maintainers, the Aliyun/Alibaba Cloud support has been deprecated. The current implementation will be left as-is until the implementation needs updates or otherwise becomes incompatible. At that point, it will be removed. We very much welcome anyone willing to contribute to this cloud provider.
```

Comment:
This should be updated in 1.23 release notes to signal the change in 1.24.

Reply:
Updated.